static/app/components/events/eventEntries.tsx

```diff
@@ -416,13 +414,16 @@ const EventEntries = ({
           }
         />
       )}
-      {!isShare && (
-        <MiniReplayView
+      {!isShare && hasEventAttachmentsFeature && (
+        <RRWebIntegration
           event={event}
-          orgFeatures={orgFeatures}
-          orgSlug={orgSlug}
-          projectSlug={projectSlug}
-          replayId={replayId}
+          orgId={orgSlug}
+          projectId={projectSlug}
+          renderer={children => (
+            <StyledReplayEventDataSection type="context-replay" title={t('Replay')}>
+              {children}
+            </StyledReplayEventDataSection>
+          )}
         />
```

Comment:
ugh, i just realized... can people get both the old rrweb integration and Replay rendered at the same time? the code before didn't let that happen.